packages/squiggle-lang/src/rescript/utility/Sparklines.res

```diff
@@ -0,0 +1,51 @@
+// Port of Sindre Sorhus' Sparkly to Rescript
+// reference implementation: https://github.com/sindresorhus/sparkly
+// Omitting rgb "fire" style, so no `chalk` dependency
+
+type sparklyConfig = {
+  minimum: option<float>,
+  maximum: option<float>
+}
+
+let sparkly = (
+  numbers: list<float>,
```

Comment:
Why not an array? We have methods in ``E.A`` for min/max for arrays. (I'm not sure what's best here_

Reply:
should've pushed right when I hit the `mark for review` button, I just got distracted by adding another test case with `Normal.add` function.